packages/flutter_tools/lib/src/usage.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
```

Comment:
ah, which explains this guy: https://travis-ci.org/flutter/flutter/jobs/244783487#L314